app/init.go

```diff
@@ -0,0 +1,135 @@
+package app
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/app"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/commands/server"
+	"github.com/iov-one/weave/crypto"
+
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+	"github.com/iov-one/weave/x/cash"
+	"github.com/iov-one/weave/x/currency"
+	"github.com/iov-one/weave/x/multisig"
+	"github.com/iov-one/weave/x/validators"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+// GenInitOptions will produce some basic options for one rich
+// account, to use for dev mode
+func GenInitOptions(args []string) (json.RawMessage, error) {
+	// Your coins ticker code
+	code := "PRJCT"
+	if len(args) > 0 {
+		code = args[0]
+	}
+
+	var addr string
+	if len(args) > 1 {
+		addr = args[1]
+	} else {
+		// if no address provided, auto-generate one
+		// and print out a recovery phrase
+		bz, phrase, err := GenerateCoinKey()
+		if err != nil {
+			return nil, err
+		}
+		addr = bz.String()
+		fmt.Println(phrase)
+	}
+
+	type (
+		dict  map[string]interface{}
+		array []interface{}
+	)
+	collectorAddr, err := hex.DecodeString("3b11c732b8fc1f09beb34031302fe2ab347c5c14")
```

Comment:
I think it is worth adding a comment what collector address is (to whom it belongs to?) and why it is hardcoded here.

Reply:
Nice point. I will take `bnsd/init` as an example because `mycoind/init` looks too much outdated